pkg/gateway/playback_test.go

```diff
@@ -45,7 +48,17 @@ var (
 	IdTranslator *testutil.UploadIDTranslator
 )
 
+func TestGatewayRecordingDB(t *testing.T) {
+	viper.Set("database.kv_enabled", false)
+	testGatewayRecording(t)
+}
+
 func TestGatewayRecording(t *testing.T) {
+	viper.Set("database.kv_enabled", true)
+	testGatewayRecording(t)
+}
```

Comment:
1. would not play with global viper configuration during test if possible - just passing an argument to testGatewayRecording if possible
2. check with @guy-har if this is still relevant or we can remove the recording parts

Reply:
Checked with @guy-har - the tests are not relevant anymore - removing it as part of this PR. (closes #2031)